File: sem-3-labs/dsal/lab3/q1.c
#include<stdio.h>
struct complex
{
	int real;
	int img;
};
void sum(struct complex a, struct complex b)
{
	int realSum,imgSum;
	realSum=a.real+b.real;
	imgSum=a.img+b.img;
	printf("The sum is %d + %di\n",realSum,imgSum);
}
void diff(struct complex a, struct complex b)
{
	int realDiff,imgDiff;
	realDiff=a.real-b.real;
	imgDiff=a.img-b.img;
	printf("The difference is %d + %di\n",realDiff,imgDiff);
}
void pro(struct complex a, struct complex b)
{
	int realPro,imgPro;
	realPro=(a.real*b.real)-(a.img*b.img);
	imgPro=(a.real*b.img)+(a.img*b.real);
	printf("The product is %d + %di\n",realPro,imgPro);
}
int main()
{
	struct complex a;
	struct complex b;
	printf("Enter first complex number: ");
	scanf("%d %d", &a.real, &a.img);
	printf("Enter second complex number: ");
	scanf("%d %d", &b.real, &b.img);
	sum(a,b);
	diff(a,b);
	pro(a,b);

}

File: sem-6-labs/itl/lab3/q1.py
a = int(input('first number: '))
b = int(input('second number: '))
c = int(input('1: addition, 2: subtraction, 3: multiplication, 4: division \n'))
if (c == 1):
    print('sum: ',(a+b))
elif(c == 2):
    print('difference: ',(a-b))
elif(c == 3):
    print('product: ',(a*b))
elif(c == 4):
    print('quotient: ',(a/b))
else:
    print('invalid')
